library_generation/test/generate_library_integration_test.sh

```diff
@@ -45,11 +45,14 @@ done
 script_dir=$(dirname "$(readlink -f "$0")")
 source "${script_dir}/../utilities.sh"
 library_generation_dir="${script_dir}"/..
-cd "${library_generation_dir}"
+output_folder="$(pwd)/output"
+mkdir -p "${output_folder}"
```

Comment:
we need external repositories to be located in there as well, which is handled by `generate_showcase.sh` and `generate_library_integration_test.sh`. I guess it would make more sense to rely on a function to prevent duplication

Reply:
Fixed in https://github.com/googleapis/sdk-platform-java/pull/1987/commits/0ebc0982892e796305216eb343098bfcae7179cf